### ISSUE TITLE
Wrong local declaration `local0` in extracted implicits

### ISSUE THREAD

**Issue by fikovnik** [PRL-PRG/scala-implicits-analysis#20]
From the following code:

```
class ExampleTest extends FunSuite with Matchers {

  test("one") {
    implicit val s1: String = Example.fun
  }

}
```

The `s1` will have fqn `local0` and escapes into extracted declarations.

**Comment by fikovnik** [comment]
```scala
  test("issue #15 - local0") {
    val local0 = idx.implicitDeclarations.find(_.declarationId == "local0")
    local0 should not be defined
  }
```

**Comment by fikovnik** [comment]
Fixed in 3978e771a8a95f2aadbc5135039efeda0a5154fe
